styles/styles.css

```diff
@@ -0,0 +1,238 @@
+* {
+    margin: 0;
+    padding: 0;
+    max-width: 100%;
+}
+
+body {
+    background-color: white;
+}
+
+header {
+    height: 50px;
+}
+
+#header_container {
+    display: flex;
+    flex-direction: row;
+    width: 100vw;
+    height: 100%;
+    border-bottom: 1px solid grey;
+}
+
+.fornavis {
+    margin: auto 18px;
+}
+
+.navis {
+    display: block;
+    color: black;
+}
+
+.forimg a {
+    text-decoration: none;
+    color: black;
+}
+
+.forimg span {
+    letter-spacing: 0.01em;
+    font: 24px youtube-sans-bold, Sans-Serif, Georgia, Arial;
+}
+
+.forimg sup {
+    font: 9px Roboto, Arial, Sans-Serif;
+}
+
+.imgspan {
+    margin: 6px -5px -10px 0px;
+}
+
+.forform {
+    margin: 5px 0px 0px 90px;
```

Comment:
Please make the indentation 2 spaces instead of 4 or tabs. And please try to use sans-serif instead of Sans-Serif. It will be ok if you use no unit for 0 unit.